k8s.tf

```diff
@@ -1,110 +1,125 @@
-    value = "${var.linuxNodeDiskSize}G"
-
-    propagate_at_launch = true
-  }
-}
-
-resource "aws_autoscaling_group_tag" "execnodes" {
-  autoscaling_group_name = data.aws_eks_node_group.execnodes.resources[0].autoscaling_groups[0].name
-
-  tag {
-    key   = "k8s.io/cluster-autoscaler/node-template/label/purpose"
-    value = "execution"
-
-    propagate_at_launch = true
-  }
-}
-
-# see https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md#auto-discovery-setup
-#     https://github.com/kubernetes/autoscaler/issues/1869#issuecomment-518530724
-resource "aws_autoscaling_group_tag" "execnodes_node-template_resources_ephemeral-storage" {
-  autoscaling_group_name = data.aws_eks_node_group.execnodes.resources[0].autoscaling_groups[0].name
-
-  tag {
-    key   = "k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage"
-    value = "${var.linuxExecutionNodeDiskSize}G"
-
-    propagate_at_launch = true
-  }
-}
-
-resource "aws_autoscaling_group_tag" "gpuexecnodes" {
-  count                  = var.gpuNodePool ? 1 : 0
-  autoscaling_group_name = data.aws_eks_node_group.gpuexecnodes[0].resources[0].autoscaling_groups[0].name
-
-  tag {
-    key   = "k8s.io/cluster-autoscaler/node-template/label/purpose"
-    value = "gpu"
-
-    propagate_at_launch = true
-  }
-}
-
-resource "aws_autoscaling_group_tag" "gpuexecnodes_node-template_resources_ephemeral-storage" {
-  count                  = var.gpuNodePool ? 1 : 0
-  autoscaling_group_name = data.aws_eks_node_group.gpuexecnodes[0].resources[0].autoscaling_groups[0].name
-
-  tag {
-    key   = "k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage"
-    value = "${var.gpuNodeDiskSize}G"
-
-    propagate_at_launch = true
-  }
-}
-
-resource "aws_autoscaling_group_tag" "gpuivsnodes" {
-  count                  = var.ivsGpuNodePool ? 1 : 0
-  autoscaling_group_name = data.aws_eks_node_group.gpuivsnodes[0].resources[0].autoscaling_groups[0].name
-
-  tag {
-    key   = "k8s.io/cluster-autoscaler/node-template/label/purpose"
-    value = "gpu"
-
-    propagate_at_launch = true
-  }
-}
+# module "eks-addons" {
+#   source                              = "git::https://github.com/aws-ia/terraform-aws-eks-blueprints.git//modules/kubernetes-addons?ref=v4.32.1"
+#   eks_cluster_id                      = module.eks.eks_cluster_id
+#   enable_aws_load_balancer_controller = false
+#   enable_aws_for_fluentbit            = var.enable_aws_for_fluentbit
+#   tags                                = var.tags
+
+#   aws_for_fluentbit_helm_config = {
+#     values = [templatefile("${path.module}/templates/fluentbit_values.yaml", {
+#       aws_region           = data.aws_region.current.name,
+#       log_group_name       = local.log_group_name,
+#       service_account_name = "aws-for-fluent-bit-sa"
+#     })]
+#     dependency_update = true
+#   }
+
+#   #depends_on                     = [module.eks.managed_node_groups]
+# }
```

Comment:
l 13 to l 30 remove, see current main branch

Reply:
Removed.